migrations/sql/V2020.10.29.09.24__add_permit_number_sequence copy.sql

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
The filename has a "copy" at the end. :P

Reply:
renamed, I will have to close and reopen the PR though